src/components/CohortBuilder/SqonBuilder/SqonBuilder.css

```diff
@@ -0,0 +1,7 @@
+.sqonBuilder .sqonListActionButton {
+  background-color: transparent;
+}
+
+.sqonBuilder .actionHeaderContainer button {
+  background-color: transparent;
+}
```

Comment:
let's do this in the `SqonBuilderContainer` styled component in `src/components/CohortBuilder/SqonBuilder/index.js` instead of a css file :)